### PR TITLE
Chore: Update nns governance did file and related

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Update did file for NNS Governance.

# Changes

* Candid file of NNS Governance
* Changes in all related files and affected types.

# Tests

* No extra test needed.

### CODE REVIEW

File: packages/nns/candid/governance.certified.idl.js [hunk 1]
```diff
@@ -629,21 +656,40 @@ export const init = ({ IDL }) => {
...
+    'status' : IDL.Opt(IDL.Int32),
+    'freezing_threshold' : IDL.Opt(IDL.Nat64),
+    'controllers' : IDL.Vec(IDL.Principal),
+    'memory_size' : IDL.Opt(IDL.Nat64),
+    'cycles' : IDL.Opt(IDL.Nat64),
+    'idle_cycles_burned_per_day' : IDL.Opt(IDL.Nat64),
+    'module_hash' : IDL.Vec(IDL.Nat8),
+  });
+  const CanisterSummary = IDL.Record({
+    'status' : IDL.Opt(CanisterStatusResultV2),
```
Comment (peterpeterparker) [inline packages/nns/candid/governance.certified.idl.js:669]: I know not related to your PR but, that's correct? Not sure what the `memory` and `cycles` balance of canister does in the governance proposal 🤷‍♂️

File: packages/nns/src/types/governance_converters.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export type Action =
   | { SetDefaultFollowees: SetDefaultFollowees }
   | { Motion: Motion }
   | { SetSnsTokenSwapOpenTimeWindow: SetSnsTokenSwapOpenTimeWindow }
-  | { OpenSnsTokenSwap: OpenSnsTokenSwap };
+  | { OpenSnsDecentralizationSale: OpenSnsDecentralizationSale };
```
Comment (peterpeterparker) [inline packages/nns/src/types/governance_converters.ts:33]: so they renamed `OpenSnsTokenSwap` but, they did not rename `SetSnsTokenSwapOpenTimeWindow`?

**Review by peterpeterparker — APPROVED** [review]
Accepting your PR. Added two comments about the NNS governance code which spontaneously seems odd to me but, only my two cents.

File: packages/nns/candid/governance.certified.idl.js (hunk 1 above)
Comment (lmuntaner) [reply packages/nns/candid/governance.certified.idl.js:669]: Good question, I also don't understand why this is needed...

File: packages/nns/src/types/governance_converters.ts (hunk 2 above)
Comment (lmuntaner) [reply packages/nns/src/types/governance_converters.ts:33]: I believe `SetSnsTokenSwapOpenTimeWindow` might be removed. Don't they want to change how the proposal starts and ends and so on?

**Comment by lmuntaner** [comment]
Rename cancelled.

**Pull request closed without merging**